airflow/contrib/hooks/gcp_dataproc_hook.py

```diff
@@ -241,6 +241,40 @@ def wait(self, operation):
         submitted = _DataProcOperation(self.get_conn(), operation)
         submitted.wait_for_done()
 
+    @staticmethod
+    def _execute_delete(service, project_id, region, cluster_name):
+        response = service.projects().regions().clusters().delete(
+            projectId=project_id,
+            region=region,
+            clusterName=cluster_name
+        ).execute()
```

Comment:
```suggestion
        ).execute(num_retries=5)
```
Google API sometimes does not respond correctly. You should send a request again.

Reply:
Great, thanks!